tools/check/bazel-check-abi.sh

```
@@ -0,0 +1,36 @@
+gosrc_md5+=("src/internal/runtime/maps/table.go 1ff4f281722eb83ac7d64ae0453e9718")
+gosrc_md5+=("src/internal/abi/map_swiss.go 7ef614406774c5be839e63aea0225b00")
+gosrc_md5+=("src/internal/abi/type.go d0caafb471a5b971854ca6426510608c")
+
+for x in "${gosrc_md5[@]}"; do
+	x=($x)
+	src="${x[0]}"
+	md5="${x[1]}"
+	echo "Checking ${src}"
+	if [ $(md5sum "${src}" | cut -d' ' -f1) != "${md5}" ]; then
+		echo "Unexpect checksum for ${src}"
+		exit -1
```

Shellcheck (0.11.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)